qemu/tests/cfg/multi_host.cfg

```diff
@@ -108,6 +108,15 @@
                     #Count of migration during file transfer.
                     migrate_count = 3
                     regain_ip_cmd = ""
+                - with_stress:
+                    only after_login_vm
+                    only mig_online
```

Comment:
suggest to limit case in mig_offline, then no need to set it for other test any more.

e.g
- mig_offline:
   only with_boot